lib/govuk_content_models/test_helpers/factories.rb

```diff
@@ -19,6 +19,13 @@
     disabled true
   end
 
+  factory :downtime do
+    message "This service will be unavailable between 3:00pm and 6:00pm tomorrow"
```

Comment:
Is this a valid example of a downtime message?  It seems somewhat brittle (ie only makes sense on the day before).